daml-lf/spec/daml-lf-1.rst

```diff
@@ -1068,6 +1104,41 @@ Then we define *well-formed expressions*. ::
     ——————————————————————————————————————————————————————————————— ExpLitNumeric
       Γ  ⊢  LitNumeric  :  'Numeric' n
 
+    ——————————————————————————————————————————————————————————————— ExpBigDecimal
+      Γ  ⊢  LitBigDecimal  :  'BigDecimal'
+
+    ——————————————————————————————————————————————————————————————— ExpMathContextUnlimited
+      Γ  ⊢  'MATHCTX_UNLIMITED'  :  'MathContext'
+
+      Γ  ⊢  r : 'RoundingMode'
+      Γ  ⊢  p : 'Int64'
+    ——————————————————————————————————————————————————————————————— ExpMathContextBounded
+      Γ  ⊢  'MATHCTX_BOUNDED' r p  :  'MathContext'
+
+    ——————————————————————————————————————————————————————————————— ExpRoundingModeCeiling
+      Γ  ⊢  'ROUNDING_CEILING'  :  'RoundingMode'
+
+    ——————————————————————————————————————————————————————————————— ExpRoundingModeFloor
+      Γ  ⊢  'ROUNDING_FLOOR'  :  'RoundingMode'
+
+    ——————————————————————————————————————————————————————————————— ExpRoundingModeDown
+      Γ  ⊢  'ROUNDING_DOWN'  :  'RoundingMode'
+
+    ——————————————————————————————————————————————————————————————— ExpRoundingModeUp
+      Γ  ⊢  'ROUNDING_UP'  :  'RoundingMode'
+
+    ——————————————————————————————————————————————————————————————— ExpRoundingModeHalf_Down
+      Γ  ⊢  'ROUNDING_HALF_DOWN'  :  'RoundingMode'
+
+    ——————————————————————————————————————————————————————————————— ExpRoundingModeHalf_Even
+      Γ  ⊢  'ROUNDING_HALF_EVEN'  :  'RoundingMode'
+
+    ——————————————————————————————————————————————————————————————— ExpRoundingModeHalf_Up
+      Γ  ⊢  'ROUNDING_HALF_UP'  :  'RoundingMode'
+
+    ——————————————————————————————————————————————————————————————— ExpRoundingModeUnnecessary
+      Γ  ⊢  'ROUNDING_UNNECESSARY'  :  'RoundingMode'
```

Comment:
If you follow my suggestion above to make these built-ins, these types should be documented in the section on rounding modes below, instead of having new typing rules.

Reply:
also changed in 5df41bfa700df47271a094aafc008b1360f196fb